pluginmgr/webapi/plugins.go

```diff
@@ -0,0 +1,26 @@
+package webapi
+
+import (
+	"github.com/iotaledger/goshimmer/plugins/webapi"
+	webapi_broadcastData "github.com/iotaledger/goshimmer/plugins/webapi/broadcastData"
```

Comment:
Maybe here, since we are already in the webapi package, we can remove the `webapi_` prefix? Wdyt?

Reply:
great point!